aptos-move/framework/aptos-framework/sources/datastructures/big_ordered_map.move

```diff
@@ -381,6 +381,15 @@ module aptos_std::big_ordered_map {
         iter.iter_borrow(self)
     }
 
+    public fun borrow_optional<K: drop + copy + store, V: copy + store>(self: &BigOrderedMap<K, V>, key: &K): Option<V> {
```

Comment:
this is not a borrow, it returns a copy of the Value, I was confused how does this compile. I'm not sure if there's naming convention for this type of operations. @igor-aptos suggestions?

Reply:
Should I rename the method to `get` instead?

Reply:
can you return &V and do the copy outside of the function and we can keep the name borrow?

Reply:
I don't know why. Move is not allowing me to return `Option<&V>`.

Reply:
oh yeah I remember that, maybe called it `get_cloned()`

Reply:
Done.